tests/test_distance_algo.py

```diff
@@ -56,3 +56,51 @@ def test_ask_command_nl(self):
     keywords_nl_level_1 = hedy.get_suggestions_for_language('nl', 1)
     closest = hedy.closest_command(invalid_command, keywords_nl_level_1)
     self.assertEqual('keyword', closest)
+
+  def test_command_en(self):
```

Comment:
Great!! Buy maybe you can make this into a parameterized test? That is a bit more aligned with the rest of the code base.